runtime/drivers/duckdb/utils.go

```diff
@@ -246,7 +246,7 @@ func fileSize(paths []string) int64 {
 }
 
 func quoteName(name string) string {
```

Comment:
May be this can be removed and  we can use functions from `Dialect` itself ?

Reply:
Done